src/App.js

```diff
@@ -1,20 +1,10 @@
-import logo from './logo.svg';
-import './App.scss';
+import './App.css';
+import Calculator from './Components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a className="App-link" href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```

Comment:
- [ ] according to the requirements the components created should be class-based components. please change this component to a class-based component.